REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.6
 PyCall
-Conda
+PyPlot
```

Comment:
Use Requires.jl?

Reply:
So it's only loaded when the plotting is called?

Reply:
So it's only loaded if the user uses PyPlot